apps/judicial-system/backend/src/app/modules/notification/notification.service.ts

```diff
@@ -1221,17 +1241,18 @@ export class NotificationService {
     return { notificationSent: false }
   }
 
-  private async sendDefenderAssignedNotifications(
+  private async shouldSendDefenderAssignedNotification(
     theCase: Case,
-  ): Promise<SendNotificationResponse> {
-    const defenderEmail = theCase.defenderEmail
-    if (!theCase.defenderNationalId || !defenderEmail) {
-      return Promise.resolve({ notificationSent: false })
-    }
-
+    defenderEmail?: string,
+  ): Promise<true | { notificationSent: boolean }> {
```

Comment:
Seems like you can get away with having the return value only `Promise<boolean>`

Reply:
😬 😬 😬 

![Screen Shot 2022-11-10 at 10 39 20](https://user-images.githubusercontent.com/3789875/201069510-52a72a39-f68f-47ca-99dd-72fd5836ceeb.png)

Reply:
Yes, we will need to change this to false. I did not see any code using this `{notificationSent: false}` object, no?

Reply:
Yea, thats correct. I've now made changes to this. Thanks 🙏